app/models/utility_hookup.rb

```diff
@@ -24,8 +25,7 @@ def name
   attribute :status, :string, default: "unavailable"
   validates :status, presence: true, inclusion: {in: %w[ready unavailable]}
 
-  attribute :old_configuration, :json, default: -> { {} }
-  validates_associated :utility
+  # validates_associated :utility
```

Comment:
I pretty sure I want to bring this back but it makes a form go sad somewhere so I need to be less stoned to figure it out.